omniscidb/L0Mgr/L0Mgr.cpp

```diff
@@ -174,6 +177,13 @@ std::unique_ptr<L0CommandList> L0Device::create_command_list() const {
   return std::make_unique<L0CommandList>(res);
 }
 
+unsigned L0Device::maxGroupCount() const {
```

Comment:
Can we use an explicitly sized type here?

Reply:
Done